tests/core/full_node/test_mempool.py

```diff
@@ -389,6 +407,19 @@ async def test_correct_block_index(self, two_nodes):
         assert status == MempoolInclusionStatus.SUCCESS
         assert err is None
 
+    @pytest.mark.asyncio
+    async def test_block_index_garbage(self, two_nodes):
```

Comment:
the `_garbage` tests ensure we ignore additional values after the condition arguments